### PR TITLE
LPS-87432 Simplify

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Julien, 
te envío esta PR para que revises lo del bundler. Gracias!

### CODE REVIEW

**Comment by julien** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/document-library/document-library-preview-pdf/src/main/resources/META-INF/resources/preview/js/PdfPreviewer.es.js [hunk 1]
```diff
@@ -55,10 +55,10 @@ class PdfPreviewer extends Component {
 
 		for (let i = 1; i <= numberOfPages; i++) {
 			if (currentPage + i <= this.totalPages) {
-				pagesUrlToLoad.push(`${this.baseImageURL}${currentPage + i}`);
+				pagesUrlToLoad.push(this.baseImageURL + (currentPage + i));
 			}
```
Comment (julien) [inline modules/apps/document-library/document-library-preview-pdf/src/main/resources/META-INF/resources/preview/js/PdfPreviewer.es.js]: @ambrinchaudhary what makes this more simple than the previous code?

**Comment by julien** [comment]
ci:test

File: modules/apps/document-library/document-library-preview-pdf/src/main/resources/META-INF/resources/preview/js/PdfPreviewer.es.js (hunk 1 above)
Comment (ambrinchaudhary) [reply modules/apps/document-library/document-library-preview-pdf/src/main/resources/META-INF/resources/preview/js/PdfPreviewer.es.js]: It is just more clear for me... what do you think about it?

Comment (julien) [reply modules/apps/document-library/document-library-preview-pdf/src/main/resources/META-INF/resources/preview/js/PdfPreviewer.es.js]: In that case I have nothing to say: I'm Ok with that. Thanks for clarifying

**Comment by boton** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**